uis/src/com/biglybt/plugin/startstoprules/defaultplugin/ui/swt/ConfigSectionDownloading.java

```diff
@@ -95,12 +95,11 @@ public Composite configSectionCreate(Composite parent) {
 	linkLabel.addMouseListener(new MouseAdapter() {
 		@Override
 		public void mouseDoubleClick(MouseEvent arg0) {
-			Utils.launch((String) ((Label) arg0.widget).getData());
+			Utils.launch(((Wiki) arg0.widget.getData()).asURL());
 		}
-
 		@Override
 		public void mouseDown(MouseEvent arg0) {
-			Utils.launch((String) ((Label) arg0.widget).getData());
+			Utils.launch(((Wiki) arg0.widget.getData()).asURL());
 		}
 	});
 	ClipboardCopy.addCopyToClipMenu( linkLabel );
```

Comment:
This looks quite similar to the default behavour in LinkLabel
```
	    ClipboardCopy.addCopyToClipMenu( label );

```

Reply:
This class will be gone/completely restructured soon.  Probably in a few days after we are sure the release went ok.

Reply:
Sweet - I'll get back to it when the changes are ready 👍